main.go

```diff
@@ -132,6 +132,18 @@ func main() {
 				Value:       ":9991",
 				Destination: &commands.VisorMetricFlags.PrometheusPort,
 			},
+			&cli.IntFlag{
+				Name:        "blockstore-cache-size",
+				EnvVars:     []string{"LILY_BLOCKSTORE_CACHE_SIZE"},
+				Value:       0,
+				Destination: &commands.CacheFlags.BlockstoreCacheSize,
+			},
+			&cli.IntFlag{
+				Name:        "statestore-cache-size",
+				EnvVars:     []string{"LILY_STATESTORE_CACHE_SIZE"},
+				Value:       0,
+				Destination: &commands.CacheFlags.StatestoreCacheSize,
+			},
```

Comment:
Could also make these Uint flags since negative values are disallowed -- error cases creating the caches could be safely ignored as well.

Reply:
You're right, could be on daemon. I can move them.

Reply:
I don't like using uint in general because Go has no overflow detection. They're best reserved for bitfields or masks. But I don't feel that strongly in this case. We use int64 for heights without a problem.

Reply:
iirc we use signed ints for height because that's the underlying type of `ChainEpoch` and specifying a negative epoch allows someone to specify a time before the genesis block. Although I don't think that's used anywhere, yet.

Reply:
Switched to uint on the flags and dependency injection structs